blog/2020-12-21-openhab-3-0-release.md

```diff
@@ -0,0 +1,89 @@
+
+An additional use case (especially considering that some users might still rely on openHAB 1.x bindings that have no equivalent in openHAB 3) is to keep an openHAB 2 instance with the legacy bindings up and running, while migrating the rest of the installation to a new openHAB 3 instance.
+
+## The New User Interface
+
+openHAB 3 introduces a state-of-the-art user interface and deprecates the existing redundant interfaces.  It is designed to cater to every user, whatever their profile, and work on all their screens, from phones to desktops, adapting its look accordingly.
+
+The new "main" UI is a radical change to what used to be on offer, but while introducing some new concepts, it stays faithful to the proven openHAB way. Things and items are still entities defined in different layers - physical and functional - and they are clearly separated and independent of each other in the new user interface.
+
+### Semantic Modelling
+
+Semantic tags were already introduced in openHAB 2, but besides the NLP-focussed HABot chat client, there wasn't yet much use for them. With the new UI, this has now changed. The semantic model lets openHAB understand what your items are representing - whether they are a location such as a room, a physical object or an interaction point. Please refer to our [new beginner tutorial](https://www.openhab.org/docs/tutorial/model.html) for a much more detailed introduction.
+
+Building a model of your home with semantically-tagged items is now recommended and can bring benefits such as cards automatically appearing on the home page, populated with relevant information.
+
+<p align="center"><img src="/uploads/2020-12_semantics.jpg"/></p>
+
+### Blockly for Automation Logic
+
+The new user interface also features a brand new way of designing your automation scripts using [Blockly](https://developers.google.com/blockly), a tool to make your program visually by assembling blocks. Just beware that your kids do not take over your home automation with it!
+
+<p align="center"><img src="/uploads/2020-12_blockly.jpg"/></p>
+
+### New Pages Concept
+
+Last but not least, the UI features new ways of interacting with your system: You can design several types of pages, which is a new concept. Types of pages include maps and floor plans that many users have asked for. Pages offer extensive features to help navigate your system. 
+
+<p align="center"><img src="/uploads/2020-12_pages.jpg"/></p>
+
+Last but not least, the new user interface is already localized in many languages - and it is very easy to help translating it to further languages. If you are interested, please visit https://translate.openhab.org/!
+
+There is so much more to learn about all the fantastic features of the new UI, so if you are interested, please do not miss to check out the presentation that was done during [our virtual openHAB meetup 2020](https://www.youtube.com/watch?v=pwZ8AOwRDEk&feature=youtu.be).
```

Comment:
```suggestion
There is so much more to learn about within the fantastic features of the new UI and if you are interested, be sure to check out the presentation that was done during [our virtual openHAB meetup 2020](https://www.youtube.com/watch?v=pwZ8AOwRDEk&feature=youtu.be).
```

Reply:
You could try the embed code, maybe it'll work:
```
<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/pwZ8AOwRDEk" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```